client/my-sites/stats/stats-post-anniversaries/single-post.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import { localize, moment } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StatsContentText from '../stats-module/content-text';
+
+const periodLabel = ( translate, period ) => {
+	if ( period.period === 'day' ) {
+		return translate( 'this day' );
+	}
+	if ( period.period === 'week' ) {
+		return translate( 'this week' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **69** times:
`translate( 'This week' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/stats/anniversary-posts)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).